Gruntfile.coffee

```diff
@@ -793,6 +793,8 @@ module.exports = (grunt) ->
 						"E032" # `.modal-content` must be a child of `.modal-dialog`
 						"W009" # Using empty spacer columns isn't necessary with Bootstrap's grid. So instead of having an empty grid column with `class="col-xs-12"` , just add `class="col-xs-offset-12"` to the next grid column.
 						"E012" # `.input-group` and `.col-*-*` cannot be used directly on the same element. Instead, nest the `.input-group` within the `.col-*-*`
+						"E017" # GCWeb wants to support explicit labels when using checkbox inputs under the .provisional > .checkbox classes
+						"E018" # GCWeb wants to support explicit labels when using radio inputs under the .provisional > .radio classes
```

Comment:
Is there a way to add a bootlint rule for explicit `labels`, to have a `for` attributes that match an `id` on an `input` element?

Reply:
You can check the bootlint website/tool: https://github.com/twbs/bootlint

Reply:
@duboisp After reading the Bootlint doc, I need to modify the `bootlint_test.js`, but can't seem to find it in `wet-boew/geweb`. Is this file located in `wet-boew/wet-boew` instead?

Reply:
Is it just a typo perhaps ?
gcweb not geweb

Reply:
Je pense que c'est cette section du gruntfile que tu recherche car c'est les paramètre envoyé à bootlint. Un fichier "coffee" est à la base un fichier javascript.

Reply:
@duboisp the coffee file adds exception the bootlint_test.js file adds rules. The coffee file in this PR already adds the exception to make sure travis doesn't complain. But I'm looking into a way to make GCWeb specific bootlint rules too when building plugins or components for Canada.ca.